content/ko/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -126,7 +126,17 @@ kubeadm 1.17 이전 버전에는 `kubeadm upgrade node` 명령에서
 
 `kubeadm certs renew` 명령을 사용하여 언제든지 인증서를 수동으로 갱신할 수 있다.
 
-이 명령은 `/etc/kubernetes/pki` 에 저장된 CA(또는 프론트 프록시 CA) 인증서와 키를 사용하여 갱신을 수행한다.
+이 명령은 `/etc/kubernetes/pki` 에 저장된 CA(또는 프론트 프록시 CA) 인증서와 키를 사용하여 갱신을 수행한다. 
+
+명령을 실행한 후에는 컨트롤 플래인 파드를 재시작해야 한다.
+현재 일부 구성 요소 및 인증서에 대해 동적 인증서를 다시 로드하는 것이 지원되지 않으므로 이 작업이 필요하다.
```

Comment:
[[원문](https://github.com/kubernetes/website/blob/main/content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md#manual-certificate-renewal)]
> After running the command you should restart the control plane Pods. This is required since dynamic certificate reload is currently not supported for all components and certificates.

```suggestion
명령을 실행한 후에는 컨트롤 플레인 파드를 재시작해야 한다.
이는 현재 일부 구성 요소 및 인증서에 대해 인증서를 동적으로 다시 로드하는 것이 지원되지 않기 때문이다.
```